server/views/partials/layout.html

```diff
@@ -33,25 +33,46 @@
 {% endblock %}
 
 {% block footer %}
-  <div class="feedback-banner govuk-!-display-none-print" data-test="feedback-banner">
-    <a id="surveyLink"
-       href=""
-       class="govuk-link govuk-link--inverse govuk-!-font-size-16"
-       target="_blank"
-       rel="noopener noreferrer"
-    >Give feedback on Digital Prison Services (opens in a new tab)</a>
+<div class="govuk-!-display-none-print">
+  <div class="govuk-width-container">
+    <div class="feedback-banner govuk-!-display-none-print govuk-width-container" data-test="feedback-banner">
+      <a id="surveyLink"
+         href=""
+         class="govuk-link govuk-link--inverse govuk-!-font-size-16"
+         target="_blank"
+         rel="noopener noreferrer"
+      >Give feedback on Digital Prison Services (opens in a new tab)</a>
+    </div>
   </div>
+</div>
+<div class="govuk-!-display-none-print">
   {{ govukFooter({
     meta: {
-      title: 'Support',
-      items: [{ text:'Support', href:'https://support.hmpps.service.justice.gov.uk/feedback-and-support' }]
-    }
+      html: '
```

Comment:
i had to do the html way because the href was not picking up the {{dpsHome}} variable. Tried combination of items and html but they were coming up in different lines on page. Hence had to move all of them into html.